context/http.go

```diff
@@ -32,12 +32,10 @@ func parseIP(ipStr string) net.IP {
 // account proxy headers.
 func RemoteAddr(r *http.Request) string {
 	if prior := r.Header.Get("X-Forwarded-For"); prior != "" {
-		proxies := strings.Split(prior, ",")
-		if len(proxies) > 0 {
-			remoteAddr := strings.Trim(proxies[0], " ")
-			if parseIP(remoteAddr) != nil {
-				return remoteAddr
-			}
+		remoteAddr, _, _ := strings.Cut(prior, ",")
+		remoteAddr = strings.Trim(remoteAddr, " ")
+		if parseIP(remoteAddr) != nil {
+			return remoteAddr
 		}
```

Comment:
> But if there's no "comma" in the header, it would be just ignored

Are you sure about this? Maybe I'm missing something but https://pkg.go.dev/strings#Split 

>  if s does not contain sep and sep is not empty, Split returns a slice of length 1 whose only element is s.

Reply:
Oh, 🙈 you're right; guess I had a mental "off by one" moment?

Reply:
so the length check was at least redundant ☺️